tests/unit/parser.js

```diff
@@ -6935,6 +6935,7 @@ exports.testStrictDirectiveASI = function (test) {
     .test("'use strict'\n(function fn() {})();", options);
 
   TestRun(test, 5)
+    .addError(2, "Missing \"use strict\" statement.")
```

Comment:
This change looks right to me, but since it's a change to an existing test, I'd like a third opinion. @caitp what do you think?